go.mod

```diff
@@ -4,9 +4,9 @@ go 1.20
 
 require (
 	github.com/containerd/log v0.1.0
-	github.com/opencontainers/image-spec v1.1.0-rc5
+	github.com/opencontainers/image-spec v1.1.0
 	github.com/stretchr/testify v1.8.4
-	golang.org/x/sys v0.10.0
+	golang.org/x/sys v0.26.0
```

Comment:
Was there a specific reason for updating this one? Actually wondering if we could drop this dependency as a whole. It looks like we only use it for `Uname()` https://github.com/containerd/platforms/blob/e6f3bbffce40d428c624961e98b4353c333a9d2a/cpuinfo_linux.go#L34


which is also provided by stdlib `syscall`; https://pkg.go.dev/syscall#Uname

If there's no difference between the two, we may as well reduce dependencies here (as its a library module, so good to reduce dependencies where possible)

WDYT?

Reply:
> Was there a specific reason for updating this one? 

No

>  It looks like we only use it for Uname()

Doesn't seem so

https://github.com/containerd/platforms/blob/e6f3bbffce40d428c624961e98b4353c333a9d2a/defaults_windows.go#L31

Reply:
ah! sorry, looked at the linux part only. Yeah, that's a pity.